[FEAT] Category 추가 Category 선택 뷰 레이아웃 구현

noah0316 opened the pull request:
## 🟣 관련 이슈

<!-- 관련있는 이슈 번호(#000)을 적어주세요. -->
- closed: #149 
## 🟣 구현/변경 사항 및 이유
Category 추가 Category 선택 뷰 레이아웃을 구현하였습니다.
<!-- 구현/변경한 내용과 그 이유를 적어주세요. -->

## 🟣 PR Point
- ImageLiteral 오타부분 수정하였습니다!
<!-- 리뷰어 분들이 집중적으로 보셨으면 하는 내용을 적어주세요 -->
## 🟣 참고 사항
| iPhone 13 mini|  	
|-----------	|
|![Simulator Screen Recording - iPhone 13 mini - 2022-01-20 at 16 09 31](https://user-images.githubusercontent.com/63908856/150291618-83cd1e3e-4dcb-419c-8ad9-72633712520b.gif)|

<!-- 참고할 사항(+스크린샷)이 있다면 적어주세요. -->

noah0316 committed:
[CORRECT] ImageLiteral 오타 수정 (#149)

noah0316 committed:
[FEAT] Category Collection View Cell 레이아웃 구현 (#149)

noah0316 committed:
[FEAT] Category 추가 Category 선택 뷰 레이아웃 구현 (#149)

noah0316 committed:
[CHORE] 프로젝트 파일 추가파일 반영 (#149)

YoonAh-dev commented on Havit/ShareExtension/Screens/Category/View/Cell/CategoryCollectionViewCell.swift:
```diff
@@ -0,0 +1,94 @@
+//  Created by Noah on 2022/01/20.
+//
+
+import UIKit
```
p1;

import Snapkit이 안들어갔어유

noah0316 replied:
SnapKit 쓸 수 있어서 깜빡했네요!! 반영했습니다!!
> 반영 commit :  [링크](https://github.com/TeamHavit/Havit-iOS/pull/162/commits/f89f39839e7f028f617ee8e7b7c3a2685451118e#diff-6b9aecfc04356ae5ad96ea55c0a88bab8aff89a9ca234b31420b764e50fce941R10)

YoonAh-dev commented on Havit/ShareExtension/Screens/Category/View/SelectCategoryViewController.swift:
```diff
@@ -0,0 +1,188 @@
+
+        var container = AttributeContainer()
+        container.font = .font(.pretendardSemibold, ofSize: 12)
+
+        configuration.attributedTitle = AttributedString("카테고리 추가", attributes: container)
+
```
p1;

공백 없애주시면 좋을 거 같아유

noah0316 replied:
반영했습니다!!
> 반영 commit : [링크](https://github.com/TeamHavit/Havit-iOS/pull/162/commits/f89f39839e7f028f617ee8e7b7c3a2685451118e#diff-778c97aae6e98da020d328a2afe0a1af882f1ca6793d095667aa9496741ebfdbL50)

noah0316 committed:
[CHORE] 코드드리뷰 반영 (#149)
- CategoryCollectionViewCell.swift: import SnapKit
- SelectCategoryViewController.swift: 공백제거

YoonAh-dev reviewed (approved):
호ㅏ이팅 꽈리~

Suyeon9911 reviewed (approved):
어풀읍 도장쾅리 ~ 읏되네요
